Polyfills/XMLHttpRequest/Source/XMLHttpRequest.h

```diff
@@ -46,3 +46,6 @@ namespace Babylon::Polyfills::Internal
         std::unordered_map<std::string, std::vector<Napi::FunctionReference>> m_eventHandlerRefs;
     };
 }
+
+bool IsHexChar(const char& c);
```

Comment:
Is this the right way to add little helper functions that are purely for internal use?

Reply:
Nope :-) If you put these functions in the header file, then they are exposed to anyone who includes them. What you should do instead is put the implementation functions at the top of the top inside the anonymous namespace.